post-payments.php

```diff
@@ -148,6 +148,11 @@ public function get_report_data( $from_date, $to_date ) {
 					'key' => $this->meta_key,
 					'compare' => 'EXISTS',
 				),
```

Comment:
I think this component of the meta key is now moot. Can you test what happens when you remove it?